packages/conda-store/src/condaStoreEnvironmentManager.ts

```diff
@@ -453,6 +453,17 @@ export class CondaStorePackageManager implements Conda.IPackageManager {
     return this.mergeConvert(installed, available);
   }
 
+  /**
+   * Boolean getter that when false means that calling loadMorePackages again
+   * should not return more packages. When true, calling loadMorePackages should
+   * fetch more packages.
+   *
+   * @return {boolean}
+   */
+  get hasMorePackages(): boolean {
```

Comment:
@peytondmurray this is probably the code that needs the most scrutiny from you because it's a complete guess on my part